### ISSUE TITLE
Drop webpack 3 support

### ISSUE THREAD

**Issue by sapegin** [styleguidist/react-styleguidist#915]
Looks like there's no way to support both at the same time and avoid deprecation warning on both versions. Let's wait for [webpack 4 support](https://github.com/facebook/create-react-app/pull/4077) in Create React App, and drop webpack 3 support in Styleguidist.

- [x] Remove conditions from [make-webpack-config.js](https://github.com/styleguidist/react-styleguidist/blob/master/scripts/make-webpack-config.js)
- [x] Update [min webpack version](https://github.com/styleguidist/react-styleguidist/blob/dfcb73de01d2b3ecf0864a95d358b216dcf65618/scripts/utils/ensureWebpack.js#L10)
- [x] Remove webpack 3 env from [Travis CI config](https://github.com/styleguidist/react-styleguidist/blob/dfcb73de01d2b3ecf0864a95d358b216dcf65618/.travis.yml#L8)
- [ ] Remove webpack conditions introduced in #1148

**Comment by nicoffee** [comment]
Hello!
I'l be happy to do it.

**Comment by jklemm** [comment]
Is this PR related to this warnings?

```
warning "react-styleguidist > react-docgen-displayname-handler@2.1.0" has incorrect peer dependency "recast@0.12.6 || 0.13.0".

warning "react-styleguidist > webpack-dev-server@2.11.2" has incorrect peer dependency "webpack@^2.2.0 || ^3.0.0".

warning "react-styleguidist > webpack-dev-server > webpack-dev-middleware@1.12.2" has incorrect peer dependency "webpack@^1.0.0 || ^2.0.0 || ^3.0.0".
```

I'm using `webpack==4.17.2`, `webpack-dev-server==3.1.9`.

**Comment by SavePointSam** [comment]
Create React App 2.0 has been released with Webpack 4 support.

**Comment by AndreyGladkov** [comment]
Remove webpack-dev-server warning https://github.com/styleguidist/react-styleguidist/pull/1180
